packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -713,3 +713,18 @@ class UiKitViewController {
     await SystemChannels.platform_views.invokeMethod<void>('dispose', id);
   }
 }
+
+/// A helper class to help implementing a new platform view.
+abstract class PlatformViewController {
+
+  /// This is invoked when the Framework is disposing the platform view. Implement this to properly dispose resources of the new platform view from the platform.
```

Comment:
The summary should probably say: `Disposes the associated platform view.`
And you should probably mention that the controller is unusable after this is called.